package.json

```diff
@@ -76,6 +77,7 @@
       "jsencrypt": "npm:jsencrypt@^2.3.1",
       "jwt-decode": "github:auth0/jwt-decode@^2.1.0",
       "lodash": "npm:lodash@^4.14.2",
+      "querystring": "github:jspm/nodelibs-querystring@^0.1.0",
```

Comment:
From Brian's comment, I guess if you can avoid using this that would be good. If you think it has better features or think `$location.search()` is insufficient you can push back.